include/minuteman.hrl

```diff
@@ -9,6 +9,12 @@
 -author("sdhillon").
 -author("Tyler Neely").
 
+-record(ct_timer, {
```

Comment:
it's used within the program, so i can add it to the ct test directly, and keep it private in the source as well, or put it somewhere shared